components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -0,0 +1,148 @@
+Create as many Data Feed instances as needed.
+
+To add a Data Feed instance in Chef Automate:
+
+![Setup Data Feed Page](/images/docs/filled_form_create_data_feed.png)
+
+1. In the **Settings** tab, navigate to _Data Feeds_ in the sidebar
+1. Select **Create Data Feed**
+1. Enter a unique Data Feed name
+1. Enter the URL for your Data Feed endpoint, including any specific port details
+1. Enter the Username and Password that your 3rd party endpoint requires for authentication
+1. Select **Test Data Feed** to begin validating the connection details
+1. Once the test is successful, select **Create Data Feed** to save your Data Feed configuration
+
+## Edit a Data Feed Instance
+
+To edit a Data Feed instance in Chef Automate:
+
+1. In _Data Feeds_, select the Data Feed name to open its detail page
+1. Edit the Data Feed name or URL
+1. Use the **Save** button to save your changes
+
+## Delete a Data Feed Instance
+
+To delete a Data Feed instance in Chef Automate:
+
+1. In _Data Feeds_, select **Delete Data Feed** from the menu at the end of the table row
+1. Select **Delete Data Feed** to confirm permanent deletion of this Data Feed
+
+## Configuring Global Data Feed Behavior
+
+{{< info >}}
+The settings in `config.toml` apply across all configured Data Feed instances.
+{{< /info >}}
+
+Modify Data Feed behavior with configuration settings in `config.toml`.
+
+1. Navigate to `/hab/svc/data-feed-service/config/config.toml` using the Chef Automate command-line tool.
+1. Change one or more configuration settings to reflect the desired global Data Feed behavior:
+
+  - Update the `feed_interval` setting to change the interval for the Data Feed collection. The default value is four hours
+  - Update the `node_batch_size` setting to change the number of sets of node data sent in each individual batch to your endpoint. The default value is 50 nodes
+  - Use the `updated_nodes_only` setting to determine what data to include in each export. The default setting is `true`, which causes the aggregation of only the *changed* data of updated nodes since the last export. Set `updated_nodes_only` to `false` and it aggregates *all* data of updated nodes since the last export
+  - To reduce the IP address range for the collected and processed node data, update the `disable_cidr_filter` setting to `false` **and** update the `cidr_filter` setting to cover the required IP address range. For example, you may wish to send only production or test node traffic
+  - Use the `accepted_status_codes` setting to define an array of HTTP status codes that the Data Feed Service will treat as `success` if returned by the 3rd party endpoint. If the status code is not in the `accepted_status_codes` list, then an error will be logged
+
+1. Apply your changes with the Chef Automate command-line tool:
+
+```bash
+    chef-automate config patch /hab/svc/data-feed-service/config/config.toml
+```
+
+### Config.toml Example
+
+```toml
+    [service]
+
+    host = "localhost"\
```

Comment:
What is the purpose of `\`? It isn't valid TOML and will cause errors.

Reply:
I have no idea, but I removed those '\' instances to resolve this issue.